src/Core/src/Platform/Tizen/SkiaGraphicsView.Tizen.cs

```diff
@@ -0,0 +1,62 @@
+#nullable disable
+using ElmSharp;
+using SkiaSharp.Views.Tizen;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Graphics.Skia;
+
+namespace Microsoft.Maui.Platform.Tizen
+{
+	// TODO. Remove it
+	// This class a temporary used, until Microsoft.Maui.Graphics.Skia.Views.SkiaGraphicsView is updated on upstream.
+	public class SkiaGraphicsView : SKCanvasView
```

Comment:
Are we missing an update or something in Maui.Graphics? I believe we added Tizen to the package?

Reply:
Yes, we're waitting for https://github.com/dotnet/Microsoft.Maui.Graphics/pull/155 to be merged. Then we can remove this code here.

Reply:
done.